src/API/OnboardingPlugins.php

```diff
@@ -416,6 +429,34 @@ public function finish_wccom_connect( $rest_request ) {
 		);
 	}
 
+	/**
+	 * Generates a cart URL for purchasing products from WooCommerce.com.
+	 * Adds the connection nonce and other connection params so extensions and themes can be automatically installed after purchase.
+	 *
+	 * @param  object $rest_request Request details.
+	 * @return array Cart URL for WooCommerce.com
+	 */
+	public function wccom_cart( $rest_request ) {
+		// Make sure the profile wizard is marked completed when we return.
+		$profile = array_merge( get_option( 'wc_onboarding_profile', array() ), array( 'completed' => true ) );
+		update_option( 'wc_onboarding_profile', $profile );
```

Comment:
This feels a bit outside the scope of this function and while unlikely, if there is an error below we'll get a completed profiler without redirecting.  Is it possible to keep the `updateProfileItems` that marks completed on the client-side?